src/configurations/digitalhealth/style/_overides.scss

```diff
@@ -1,9 +1,23 @@
 $primary-text-color: #245879;
 $primary-action-color: #245879;
 
-#header,
+#footer,
+#header {
+  height: fit-content;
+  background-image: none !important;
+}
+
+#header {
+  position: relative;
+  .container-fluid {
+    position: absolute;
+    top: 20%;
+    width: 80%;
+  }
+}
+
 #footer {
-  background-blend-mode: multiply;
+  background-color: #3e7293 !important;
```

Comment:
It's great to remove the blend mode css. I can't think of a time to use it. Do you think we can apply the new background color without using !important? Like making the selector more specific might just do the trick. I am worried that !important is often harder for others to override this selector later on if the footer needs to be customized somehow.

Reply:
Removed the important tag, good call!